scripts/actions/check-for-outdated-translations.js

```diff
@@ -85,4 +103,6 @@ const main = async () => {
   }
 };
 
-module.exports = main;
+main();
```

Comment:
i think we should wrap this in an if check similar to how we do in other files. otherwise, this executes automatically when you import which i don't think we want.

```js
if (require.main === module) {
  main();
}
```

Reply:
done. and updated test to check for console.log being called 0 times